.github/workflows/cd-publish-packages.yml

```
@@ -0,0 +1,56 @@
+name: CD - FAST Tooling Publisher
```

For consistency with FAST

```suggestion
name: CD - FAST Tooling Publish
```